x-pack/plugins/index_management/public/application/index.tsx

```diff
@@ -44,17 +48,19 @@ export const renderApp = (
 
   render(
     <I18nContext>
-      <Provider store={indexManagementStore(services)}>
-        <AppContextProvider value={dependencies}>
-          <MappingsEditorProvider>
-            <ComponentTemplatesProvider value={componentTemplateProviderValues}>
-              <GlobalFlyoutProvider>
-                <App history={history} />
-              </GlobalFlyoutProvider>
-            </ComponentTemplatesProvider>
-          </MappingsEditorProvider>
-        </AppContextProvider>
-      </Provider>
+      <KibanaReactContextProvider>
```

Comment:
I might be missing something here, but is this necessary? I don't see it being used anywhere.

Reply:
The `CodeEditor` component used for editing the painless script requires it to provide `uiSettings`, which is really unclear from the code. I'll add a comment.

Reply:
Ah. Yeah, it isn't very clear, but I see that now after looking at the code_editor code. Thanks for adding the comment!